configure

```diff
@@ -297,7 +296,8 @@ auto_downloads = nodedownload.parse(options.download_list)
 
 
 def warn(msg):
-  prefix = '\033[1m\033[91mWARNING\033[0m' if os.isatty(1) else 'WARNING'
+  warn.warned = True
+  prefix = '\033[1m\033[93mWARNING\033[0m' if os.isatty(1) else 'WARNING'
   print('%s: %s' % (prefix, msg))
```

Comment:
You need to define `warn.warned`, else you get an AttributeError below when warn() hasn't been called.

On the topic of the commit log, can you make the first line fit in <= 50 characters?  Thanks!